CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 All notable changes to this project are documented in this file.
 
+## 0.24.1
+
+**Release date:** 2022-04-22
+
+This prerelease fixes a regression bug where the controller would panic in
+further to be identified edge-case scenarios in which a `HelmRepository`
+Artifact would not have a size.
+
+Fixes:
+- Fix panic when HelmRepository's artifact size is ni
```

Comment:
```suggestion
- Fix panic when HelmRepository's artifact size is nil
```

Reply:
This would have been perfect incorrect Dutch for "is not"